torch/onnx/_internal/exporter.py

```diff
@@ -954,10 +954,22 @@ def export(self) -> ExportOutput:
                 op_level_debug=self.options.op_level_debug,
             )
 
+            # NOTE: Filter out the initializers with fake tensors when it's fake_mode exporting,
+            # otherwise, the ONNX exporter will fail: RuntimeError: basic_string::_M_construct null
+            # not valid
```

Comment:
```suggestion
            # NOTE: Filter out the initializers with fake tensors when it's fake_mode exporting.
            # Otherwise, the ONNX exporter will fail: RuntimeError: basic_string::_M_construct null
            # not valid.
            # Concrete data is expected to be filled for those initializers later during `ExportOutput.save`.
```